cypress/integration/tests/developer/applicationinventory/assessment/filter.test.ts

```diff
@@ -150,21 +162,13 @@ describe("Application inventory filter validations", { tags: "@tier2" }, functio
 
         // Enter an existing businessservice and assert
         var validSearchInput = applicationsList[0].business;
-        applySearchFilter(businessService, validSearchInput);
+        selectFilter(businessService);
+        cy.get("div.pf-c-select").eq(1).click();
+        cy.get("ul.pf-c-select__menu").contains(validSearchInput).click();
```

Comment:
why applysearchfilter() method is not working here ?
Modify the method applySearchfilter so that it can be used for BS and tags both.

Reply:
The locators used for line 265 of utils.ts are different for Tags and BS 
BS needs "ul.pf-c-select__menu" and Tags uses "div.pf-c-select__menu"
Will still try to modify/way to incorporate both locators in the code.